renderers/photoaudiohaptics.schema.json

```diff
@@ -25,12 +25,18 @@
                 "maximum": 1
             }
         },
-        "contourPoints": {
+        "contours": {
             "type": "array",
             "description": "An array containing an array of normalized coordinates representing a point on the segment or point on the bounding box.",
             "items": {
                 "$ref": "#/definitions/coordinates"
             }
+        },
+        
+        "entityType":{
+            "type": "string",
+            "description": "This describes whether the entity is a segment, object or static text",
```

Comment:
Based on the PR with the handler code, it looks like there are only a few values for this. Could you change the type to an enumeration like below?
https://github.com/Shared-Reality-Lab/IMAGE-server/blob/ff3453c2d4974575f0f6b6681232184462c5b0e5/preprocessors/content-categoriser.schema.json#L8

Reply:
yes, that would make sense!